docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -85,10 +98,10 @@ through SilverStripe, so is considered a second line of defence.
 
 Files can be kept in draft stage,
 and access restricted to certain user groups.
-These files are stored in a special `.protected` folder (defaulting to `public/assets/.protected`).
+These files are stored in a special `.protected/` folder (defaulting to `public/assets/.protected`).
```

Comment:
Should the default value in the brackets have the trailing slash as well?

Reply:
Have added, thanks.